torch/onnx/symbolic_opset9.py

```diff
@@ -2159,6 +2159,29 @@ def pixel_shuffle(g, self, upscale_factor):
                                     allowzero=0)
 
 
+@parse_args("v", "i")
+def pixel_unshuffle(g, self, downscale_factor):
+    dims = sym_help._get_tensor_sizes(self)
+    if len(dims) != 4:
+        return _unimplemented("pixel_shuffle", "only support 4d input")
+    if any([i is None for i in dims[1:]]):
+        return _unimplemented("pixel_shuffle", "only support static input shape, except for batch size")
+    output_channel = dims[1] * downscale_factor * downscale_factor
+    after_view = sym_help._reshape_helper(g, self,
+                                          g.op("Constant", value_t=torch.tensor([-1, dims[1],
+                                                                                dims[2] // downscale_factor,
```

Comment:
are dims[2] and dims[3] guaranteed to be divisible by downscale_factor?

Reply:
Yup the shape of the input is 
(*, C, dims[2] * downscale_factor,  dims[3] * downscale_factor)
So should be always divisible